.gitignore

```diff
@@ -7,3 +7,4 @@ terraform.tfvars
 .terraform.tfstate.lock.info
 .DS_Store
 sas_iac_buildinfo.yaml
+tf-plan.out
```

Comment:
I also don't see this in any part of the code and/or doc this update should be removed.

Reply:
In the file, TerraformUsage.md, I added an optional command which seems to be a good practice, to output the tf plan to a file.  This file was added to .gitignore so that the new file did not prompt a git change:
  terraform plan
  # alternative to store your plan for later
  terraform plan -out=tf-plan.out

Reply:
Ok, I am not sure we even need the section on using `terraform plan` this is the only place it's mentioned. `terraform apply` provides the user an intervention step if needed and the ability to check. Looking over terraform docs now to see what their best practices are for this one.

Reply:
Resolved with Remove tf plan out from gitignore and TF usage md / 7d48362